iati/tests/about_functional_tests.py

```diff
@@ -1,27 +1,65 @@
-"""A module of functional tests for the about page and its sub pages."""
-from django.utils.text import slugify
-import pytest
+"""A module of functional tests for the about page and its sub pages.
 
+TODO:
+    Refactor most of these tests out into base functional tests.
 
-ABOUT_SUB_PAGE = {'page_type': 'About sub page', 'title': 'test sub page'}
-CASE_STUDY_INDEX_PAGE = {'page_type': 'Case study index page', 'title': 'test case study index page'}
-HISTORY_PAGE = {'page_type': 'History page', 'title': 'test history page'}
-PEOPLE_PAGE = {'page_type': 'People page', 'title': 'test people page'}
+"""
+from django.utils.text import slugify
+import pytest
 
 
-def navigate_to_about_cms(admin_browser):
+ABOUT_PAGE = {
+    'title': 'About',
+    'heading': 'Test About Heading',
+    'excerpt': 'This is an excerpt for the About page'
+}
+ABOUT_SUB_PAGE = {
+    'page_type': 'About sub page',
+    'title': 'test sub page',
+    'heading': 'Test Sub Page',
+    'excerpt': 'This is an excerpt for an About Sub page'
+}
+CASE_STUDY_INDEX_PAGE = {
+    'page_type': 'Case study index page',
+    'title': 'test case study index page',
+    'heading': 'Test Case Study Index Page',
+    'excerpt': 'This is an excerpt for a Case Study Index page'
+}
+HISTORY_PAGE = {
+    'page_type': 'History page',
+    'title': 'test history page',
+    'heading': 'Test History Page',
+    'excerpt': 'This is an excerpt for the History page'
+}
+PEOPLE_PAGE = {
+    'page_type': 'People page',
+    'title': 'test people page',
+    'heading': 'Test People Page',
+    'excerpt': 'This is an excerpt for the People page'
+}
+CASE_STUDY_PAGE = {
+    'page_type': 'Case study page',
+    'title': 'test case study page',
+    'heading': 'Test Case Study Page',
+    'excerpt': 'This is an excerpt for a Case Study page.'
+}
+
+H2 = {'content': 'H2 heading', 'button': 'H2', 'id': 'content_editor_en-{}-value'}
+H3 = {'content': 'H3 heading', 'button': 'H3', 'id': 'content_editor_en-{}-value'}
+H4 = {'content': 'H4 heading', 'button': 'H4', 'id': 'content_editor_en-{}-value'}
+
+
+def navigate_to_default_page_cms_section(admin_browser, default_page_title):
     """Navigate to the About page section of the CMS."""
     admin_browser.click_link_by_text('Pages')
     admin_browser.find_by_xpath('//span[@class="icon icon-arrow-right "]').click()
-    admin_browser.find_by_text('About').click()
+    admin_browser.find_by_text(default_page_title).click()
 
 
-def enter_page_content(admin_browser, page_type, page_title):
+def enter_page_content(admin_browser, tab_name, cms_field, cms_content):
```

Comment:
I do think that it's worth adding extra documentation to functions about the expected parameter types and what they related to. Here I'm guessing that `cms_content` is a string but I don't know if it related to a CSS class, ID for form name...